xiv_stats.php

```diff
@@ -481,9 +486,11 @@ function handleClass($row, $classDef, &$totalArray) {
           <div class="col s12 m6" style="width:100%;">
               <div class="card">
                   <div class="card-content">
-                      <a id="population"><span class="card-title" style="font-size:28pt;">XIVCensus - Character statistics for FFXIV</span></a>
+                      <a id="population"><span class="card-title" style="font-size:28pt;"><img src="img/logo.png" class="logo" title="XIVCensus - Character statistics for FFXIV"/></span></a>                 
+                      <img src="img/lofo.png" />
```

Comment:
Should that be "logo" rather than "lofo"? 😉

Reply:
https://github.com/XIVStats/XIVStats/pull/46/commits/5667f1393631499282efdf3bf017b1b263ba3e9d